Products/PDBDebugMode/runcall.py

```diff
@@ -3,16 +3,9 @@
 except:
     import pdb
 
-from ZPublisher import Publish
+from ZServer.ZPublisher import Publish
```

Comment:
You either need to keep the imports backwards-compatible or do a major version bump.

Reply:
done - isn't it?

merge and release?